components/eam/src/chemistry/mozart/lin_strat_chem.F90

```diff
@@ -188,6 +188,12 @@ subroutine lin_strat_chem_inti(phys_state)
 !     
 !   initialize the linoz data
 
+    !!set true for linoz_v3 interpolation in interpolation
+    if (linoz_v3) file%linoz_v3=.true.
+    if (linoz_v2) file%linoz_v2=.true.
+    if (linoz_v2.eq..true. .and.linoz_v3.eq..true..or.&
+        linoz_v2.eq..false..and.linoz_v3.eq..false.) return
```

Comment:
@jinboxie , it would be better separate these two conditions (1. both are true; 2. both are false) and print out a message for each condition in the log file at the first time step. This will provide useful information about the simulation settings.

Reply:
Updated.

Reply:
Maybe you didn't push the commit?

Reply:
Hi Qi. It is pushed again.